go/ct/common/revisions_test.go

```diff
@@ -85,3 +86,27 @@ func TestRevision_GetRevisionForBlock(t *testing.T) {
 	}
 
 }
+
+func TestRevisions_GetForkTime(t *testing.T) {
+	tests := map[tosca.Revision]struct {
+		revision tosca.Revision
+		forkTime uint64
+	}{
```

Comment:
what I ment with using the revision as the key of the map is that you can also remove it from the struct. Why would you list it twice.

All you need is a `map[tosca.Revision]uint64` mapping revisions to fork times.